kolomiets/nicofog/src/main/java/entity/Cigarette.java

```diff
@@ -1,21 +1,32 @@
 package entity;
 
+import javax.persistence.*;
 import java.time.LocalDateTime;
 
 /**
  * Represent depend one to one from user.
  * Make extend function for issue by cigarette of user
  */
+@Entity
+@Table(name = "cigarette")
 public class Cigarette {
     public Cigarette() {
     }
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
-    private LocalDateTime lastSmokeTime;
+    @Column(name = "last_smoke_time")
+    private String lastSmokeTime;
+    @Column(name = "average_time")
     private int averageTime;
```

Comment:
I think these column's have the same name already, so don't need this annotation